client/routes/search.js

```diff
@@ -1,13 +1,15 @@
 const log = require('../logger');
 const makeRequest = require('../makeRequest');
 const sendPacket = require('../sendPacket');
+const clickLogger = require('../clickLogger');
 
 const { User, Query } = require('../mongoConfig');
 
 module.exports = (app) => {
   app.get('/search/:query', async (request, response) => {
     const query = request.params['query'];
     log('query', `Received query from client: ${query}. Sending data to ranker`);
+    clickLogger.setCurrQuery(query);
```

Comment:
I dont think this current method of saving the query will work with more than 1 active user. Theres only 1 current query. I think the easier way to do this would just be to pass in query when clicking updateHistory via the route. I'll explain how to do this over call tomorrow.